recipes/network-wrangler/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "network-wrangler" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/network_wrangler-{{ version }}.tar.gz
+  sha256: 16d81335190931a2da0a1ab01f3097a59449264124c5b06ee1d1ddb1c36f71dd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - setuptools-scm
+    - wheel
+    - pip
+  run:
+    - python >=3.6
+    - geographiclib
+    - geojson
+    - geopandas >=0.8.0
+    - osmnx >=0.12
+    - pandas >=1.6
+    - pandera
+    - projectcard
```

Comment:
it seems it is missing this dependency ``projectcard``
you will need to package that first